panda/src/pgraphnodes/sceneGraphAnalyzer.I

```diff
@@ -229,7 +229,6 @@ get_texture_bytes() const {
   return _texture_bytes;
 }
 
-
```

Comment:
This whitespace change is unnecessary and is the only change to this file. It should be removed to make the PR more concise.